index.html

```diff
@@ -12,26 +12,114 @@
     <title>William Raga's Portfolio</title>
   </head>
   <body>
-    <header>
+    <header class="header-container">
       <a href="" class="white text-normal align-center roboto logo"
         >William Raga</a
       >
-      <nav class="flex-container-d">
-        <i class="fa-solid fa-bars fa-2x white hamburger-menu"></i>
-        <ul class="flex-container-d lists">
-          <li class="roboto white"><a href="" class="white">Portfolio</a></li>
-          <li class="roboto white"><a href="" class="white">About</a></li>
-          <li class="roboto white"><a href="" class="white">Contact</a></li>
-          <li class="roboto white">
-            <a href="" class="white"
+      <div
+        aria-controls="primary-navigation"
+        aria-expanded="false"
+        class="header-container__nav-list-item-burger burger mobile-nav-toggle"
+        role="navigation"
+      >
+        <div class="burger-menu rotate"></div>
+        <div class="burger-menu dissappear"></div>
+        <div class="burger-menu rotate1"></div>
+      </div>
+      <nav class="flex-container-d header-container__nav">
+
+        <!-- <i class="fa-solid fa-bars fa-2x white hamburger-menu"></i> -->
```

Comment:
- [ ] According to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) point number 12, you are supposed to remove all commented unwanted pieces of code from your code base. Having them in production code clutters the production code and is generally bad practice.